curator/patterns/hero-post.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Title: Hero Post
```

Comment:
If the latest post has no text (and thus no excerpt), the spacing becomes very tight:
 
<img width="1729" alt="Screen Shot 2022-06-14 at 11 34 10 AM" src="https://user-images.githubusercontent.com/5375500/173617974-57846e1a-d195-42c1-bbb6-02c46e6251f4.png">

When there is an excerpt to display, I think the font size needs to be bigger (32px from the comps), and the width somehow limited. Also the heading size is 170px in the comps but maybe that is too big because post titles can be long?  

Current | Intended
--- | ---
<img width="1440" alt="Screen Shot 2022-06-14 at 11 37 33 AM" src="https://user-images.githubusercontent.com/5375500/173618576-bea33e6b-890c-4966-8250-70e7294e52d8.png"> | <img width="1188" alt="Screen Shot 2022-06-14 at 11 39 17 AM" src="https://user-images.githubusercontent.com/5375500/173618854-c860e422-2f03-4a84-829f-4500e7e1f684.png">

Reply:
Without excerpt content

<img width="639" alt="image" src="https://user-images.githubusercontent.com/1935113/173775742-7896f969-9756-49c8-90d4-1a38f92b115f.png">

With excerpt content

<img width="639" alt="image" src="https://user-images.githubusercontent.com/1935113/173777391-5c7f3153-54f5-4834-a55f-faad93f4b7b4.png">

Post title is given font size of 8rem (128px). 170px appears to be too big and it breaks even the medium length post titles into multiple lines).
Excerpt is: 2rem (32px) and it takes a max width of 900px.

comps suggests them to be variable sizes. However, clamp values in the template doesn't work with GB enabled. 
But they work when GB disabled 😵‍💫 .
I will have to check if there is a related issue. Otherwise I will create one.

Reply:
created: https://github.com/WordPress/gutenberg/issues/41859